metainfo.xml

```diff
@@ -232,6 +234,53 @@
           </configuration-dependencies>
         </component>
 
+        <component>
+          <!-- CDAP Auth Server -->
+          <name>CDAP_AUTH_SERVER</name>
+          <displayName>CDAP Auth Server</displayName>
+          <category>SLAVE</category>
```

Comment:
Copied from HDFS/JOURNALNODE... I honestly don't know other than I couldn't find an example of a MASTER that was optional (cardinality 0+)